.travis.yml

```diff
@@ -1,15 +1,19 @@
 language: python
 python:
 - '2.7'
+env:
+- TOXENV=django18
+- TOXENV=django110
+- TOXENV=django111
+matrix:
+  allow_failures:
+  - env: TOXENV=django110
```

Comment:
Is this necessary? In other words, do failures actually exist when testing against Django 1.10?